src/components/PopUp.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from "react";
+import { View, StyleSheet, ViewStyle, Modal, Alert } from "react-native";
+import type { StyleProp } from "react-native";
+import { PRIMARY_COLOR, TEXT_COLOR } from "../util/colors";
+import { Button } from "./Button";
+
+export interface ModalProps {
+	text: string;
+	accessibilityLabel: string;
+	accessibilityHint?: string;
+	style?: StyleProp<ViewStyle>;
+}
+
+export const PopUp: React.FC<ModalProps> = ({
+	text,
+	accessibilityLabel,
+	accessibilityHint,
+	style,
+	children,
+}) => {
+	const [modalVisible, setModalVisible] = useState(false);
+	return (
+		<View
+			accessibilityLabel="A Pop Up"
+			accessibilityHint="A Pop Up"
```

Comment:
I think we should put some more than just "a pop up" . What's the pop-up here for?

Reply:
I designed this component to serve a broader purpose. We can use this for help text, but we can also use it in other places as well. I wasn't really sure what to put here since my idea was to make this more of a general component. I can make these fields props that we pass in if we want these to be more descriptive, lmk what you think @carpenterd777

Reply:
yeah, for a reusable component I would def prefer making the accessibility fields props @elenirotsides